APIs/schemas/input.json

```
@@ -42,6 +43,11 @@
               "type": "string"
             }
           }
+        },
+        "device_id": {
+          "description": "Device ID which this Input forms part of. This attribute is used to ensure referential integrity by registry implementations.",
```

I don't think this is related to registry implementations, so unless we can think of better explanation, I suggest deleting second sentence.

```suggestion
          "description": "Device ID which this Input forms part of",
```